### PR TITLE
Add Random player

### PR DESCRIPTION
**Pull request by FarzanRashid**
This PR adds random player to the roulette game by implement PlayerRandom class. This PR also includes tests for the PlayersRandom class.

### CODE REVIEW

File: src/players/random.py [hunk 1]
```diff
@@ -38,5 +38,7 @@ def placeBets(self) -> None:
         Updates the :py:class:`~table.Table` object with a randomly placed :py:class:`~bet.Bet`
         instance.
         """
-
-        self.table.placeBet(Bet(1, self.rng.choice(list(self.all_OC))))
+        if super().playing() and self.stake > 0:
```
Comment (FarsanRashid) [inline src/players/random.py:41]: need to provide a `playing` method. If checked from inside `placeBets`, the sessions will always run all cycles.

File: src/players/random.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        class.
+
+        :param table: table (Table) – The :py:class:`~table.Table` object which will
+        accept the bets.
+        :param wheel: wheel (wheel) - The :py:class:`~wheel.Wheel` object which will be used to
+        populate :py:class:`~outcome.Outcome`s.
+        """
+
+        super().__init__(table)
+        self.wheel = wheel
```
Comment (FarsanRashid) [inline src/players/random.py:31]: `wheel` does not need to be an attribute

File: src/players/random.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+from bet import Bet
+from players.player import Player
+
+
+class PlayerRandom(Player):
+    """
+    :class:`PlayerRandom` is a :py:class:`~players.player.Player` who places bets in Roulette.
+    This player makes random bets around the layout.
+
+    .. attribute:: rng
```
Comment (FarsanRashid) [inline src/players/random.py:11]: need docstring for `all_OC` attribute

File: src/players/random.py (hunk 2 above)
Comment (FarzanRashid) [reply src/players/random.py:31]: Removed.

File: src/players/random.py (hunk 1 above)
Comment (FarzanRashid) [reply src/players/random.py:41]: Done.

**Pull request merged**